### PR TITLE
MB 1801-Typography

### PR DESCRIPTION
**Pull request by Shkeating**
## Description

The main task in this story is to add a reference page to our Storybook for Typography. Unlike most of our Storybook contributions, we are not establishing a new component or SASS module. Changes can be found in our `index.stories.jsx` file and `_typography.scss`.

## Reviewer Notes

As mentioned in today's standup, some of the font sizes and line heights are not going to match the specs I was given because they do not match up with the provided USWDS tokens. Right now, our h3 is at `22px` because `21px `doesn't have a USWDS font token. If we want to use a different font token instead, [take a look at what is available](https://designsystem.digital.gov/design-tokens/typesetting/font-size/) and comment what you'd prefer. the same applies to our line heights which are currently as follows:

## Setup
Setup and start storybook

```sh
make client_deps
make storybook
```

Run the loki tests
```sh
make storybook_tests
```

View the Typography Page in browser http://localhost:6006/?path=/story/global-styles-typography--headers

## Code Review Verification Steps
- [x]  There are no aXe warnings for UI.
- [x] This works in Supported Browsers and their phone views (Chrome, Firefox, IE, Edge).
- [x] User facing changes have been reviewed by design.
- [x] Request review from a member of a different team.
- [x] Have the Jira acceptance criteria been met for this change?

## References

* [JIRA Ticket  MB 1801-Typography](https://dp3.atlassian.net/browse/MB-1801?atlOrigin=eyJpIjoiMTA1ODhhNjQ0ODY2NDNlZTk1YTIwZTliYmVkZTA4N2UiLCJwIjoiaiJ9)
* [Abstract Sticker Sheet](https://app.abstract.com/projects/1f6c1fd0-88a2-11e9-aaf7-a7ce5896598b/branches/master/commits/cf3af41f8d382b6307515a97d6669f440c2a22e5/files/4d7d9c92-16be-4191-868f-24205abf9e48/layers/221D42D3-D301-4BC3-A041-4E1447B0E274?collectionId=f96ae095-14cb-4c1b-bf79-0bbd4c47a3e2&collectionLayerId=c59faa6c-319f-43db-9fdb-facdc3f29909&mode=build&selected=2096354891-29270A62-4FCB-4821-B187-5AA4F33AA9F1) 
* [USWDS Line Height Tokens](https://designsystem.digital.gov/design-tokens/typesetting/line-height/)
* [USWDS Font Size Tokens](https://designsystem.digital.gov/design-tokens/typesetting/font-size/)

## Screenshots

Storybook Typography page

### CODE REVIEW

File: src/shared/styles/_typography.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  @include u-text('normal');
+}
+p {
+  @include typeset('body',  4, 2);
+}
+small {
+  @include typeset('body',  2, 2);
+}
+a {
+  text-decoration: none;
```
Comment (gidjin) [inline src/shared/styles/_typography.scss]: Does using the uswds mixin here do more than just set the decoration? For example how is this different than `u-text('none')` or `u-text('underline')`?

File: src/shared/styles/_typography.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  &:hover , &.hover {
+    text-decoration: underline;
+  }
+  &:visited, &.visted {
```
Comment (gidjin) [inline src/shared/styles/_typography.scss]: I found a typo should be `&:visited, &.visited`; so easy for me to make these too :)

File: src/shared/styles/_typography.scss (hunk 1 above)
Comment (Shkeating) [reply src/shared/styles/_typography.scss]: it's not! i just never found it in the documentation. Thanks!

**Comment by samadden** [comment]
Same feedback for h6 -- recommend increasing the line-height on h6 to the next larger token (equivalent to 1.35)

**Comment by samadden** [comment]
Recommend changing line-height on `p-small` to be `token 3` (1.35).
Screen Shot 2020-03-06 at 8 54 50 AM

**Comment by samadden** [comment]
Recommend changing line-height for `p` to be `token 4` (1.5).

Screenshot shows what `p` and `p-small` after browser editing to implement the line-heights for  tokens 4 and 3, respectively.
Screen Shot 2020-03-06 at 8 58 25 AM

**Comment by samadden** [comment]
On the Text section, the p-small is showing `line-height 1.6` (`token 5`?), and should be` token 3` (`1.35`)

**Comment by samadden** [comment]
I'm not sure if I understand how the line-height is calculated relative to the font size for `p-small` -- the line height seems tall; it is 1.4 em and the font size is .87rem. Do we need to use a smaller line height token (2?) to get the proportions closer? LMK if it's helpful to pair on this -- I want to learn how to best specify the tokens.


Screen Shot 2020-03-06 at 2 25 11 PM

**Pull request merged**